enterprise/frontend/test/metabase-enterprise/sandboxes/sandboxes.cy.spec.js

```diff
@@ -129,158 +63,146 @@ describeWithToken("formatting > sandboxes", () => {
       cy.findByText("Update").click();
+    });
 
-      // New user
-      cy.visit("/admin/people");
+    it("should add key attributes to a new user", () => {
       cy.findByText("Add someone").click();
-      cy.findByPlaceholderText("Johnny").type(new_user.first_name);
-      cy.findByPlaceholderText("Appleseed").type(new_user.last_name);
+      cy.findByPlaceholderText("Johnny").type(sandboxed_user.first_name);
+      cy.findByPlaceholderText("Appleseed").type(sandboxed_user.last_name);
       cy.findByPlaceholderText("youlooknicetoday@email.com").type(
-        new_user.username,
+        sandboxed_user.email,
       );
       cy.findByText("Add an attribute").click();
       cy.findByPlaceholderText("Key").type("User ID");
       cy.findByPlaceholderText("Value").type("1");
       cy.findAllByText("Create").click();
       cy.findByText("Done").click();
     });
+  });
 
-    it("should change sandbox permissions as admin", () => {
-      signOut();
+  describe("normal user", () => {
+    const USER_ATTRIBUTE = "User ID";
+    const ATTRIBUTE_VALUE = "3";
+    const TTAG_NAME = "cid";
+    const QUESTION_NAME = "Joined test";
+
+    beforeEach(() => {
+      restore();
       signInAsAdmin();
 
-      // Changes Orders permssions to use filter and People to use SQL filter
+      // Add user attribute to existing ("normal" / id:2) user
+      cy.request("PUT", "/api/user/2", {
+        login_attributes: { [USER_ATTRIBUTE]: ATTRIBUTE_VALUE },
+      });
+
+      // Orders join Products
+      createJoinedQuestion(QUESTION_NAME);
+
+      // Sandbox Orders table on "User ID"
       cy.request("POST", "/api/mt/gtap", {
-        id: 1,
         group_id: DATA_GROUP,
         table_id: ORDERS_ID,
         card_id: null,
         attribute_remappings: {
-          "User ID": ["dimension", ["field-id", ORDERS.USER_ID]],
+          [USER_ATTRIBUTE]: ["dimension", ["field-id", ORDERS.USER_ID]],
         },
       });
-      cy.request("POST", "/api/mt/gtap", {
-        group_id: DATA_GROUP,
-        table_id: PEOPLE_ID,
-        card_id: 4,
-        attribute_remappings: {
-          "User ID": ["dimension", ["template-tag", "cid"]],
-        },
-      });
-      cy.request("PUT", "/api/permissions/graph", {
-        revision: 1,
-        groups: {
-          [ADMIN_GROUP]: { "1": { native: "write", schemas: "all" } },
-          [DATA_GROUP]: {
-            "1": {
-              schemas: {
-                PUBLIC: {
-                  [ORDERS_ID]: { query: "segmented", read: "all" },
-                  [PEOPLE_ID]: { query: "segmented", read: "all" },
-                  [PRODUCTS_ID]: "all",
-                  [REVIEWS_ID]: "all",
-                },
+
+      cy.log("**--Create parametrized SQL question--**");
+      cy.request("POST", "/api/card", {
+        name: "sql param",
+        dataset_query: {
+          type: "native",
+          native: {
+            query: "select id,name,address,email from people where {{cid}}",
```

Comment:
Didn't miss it before, just skipped it, but would this work or would it explode because of the extra clauses?
```suggestion
            query: `select id,name,address,email from people where {{${TTAG_NAME}}}`,
```

Reply:
Great catch! It's working :)